src/schemas.py

```diff
@@ -0,0 +1,71 @@
+from pydantic import BaseModel, UUID4
+from typing import List, Any
+
+
+# Users
+class UserBase(BaseModel):
+    address: str
+
+class User(UserBase):
+  id: UUID4
+  name: str
+
+class UserCreation(UserBase):
+    name: str
+
+# Entrypoints
+class EntrypointBase(BaseModel):
+    name: str
+    contract_id: UUID4
+    is_enabled: bool
+
+class Entrypoint(EntrypointBase):
+  id: UUID4
+
+class EntrypointCreation(BaseModel):
+  name: str
+  is_enabled: bool
+
+class EntrypointUpdate(BaseModel):
+  id: UUID4
+  is_enabled: bool
+
+# Contracts
+class ContractBase(BaseModel):
+  address: str
+  owner_id: UUID4
+
+class Contract(ContractBase):
+  id: UUID4
+  name: str
+  entrypoints: List[Entrypoint]
+
+class ContractCreation(ContractBase):
+  name: str
+  entrypoints: List[EntrypointCreation]
+
+
+# Operations
+# TODO: right now the sender isn't checked, as we use permits anyway
+class CallData(BaseModel):
+    sender: str
+    operations: list[dict[str, Any]]
+
+class CreateOperation(BaseModel):
+  #  amount: int
+  contract_id: str
+  entrypoint_id: str
+  #  transaction_hash: str
+
+class UpdateHashOperation(BaseModel):
+  transaction_hash: str
+
```

Comment:
These are information we can already get from an indexer, so I'd suggest we remove them. It's your call though.

Also what is the second class?

Reply:
I don't understand well what do you mean by ` we can already get from an indexer` so maybe discuss to this in a call

> Also what is the second class?

I think it's a dead code, I'll remove it

Reply:
> I don't understand well what do you mean by  we can already get from an indexer so maybe discuss to this in a call

I was confused about those, sorry.